### PR TITLE
SEARCH-402/403 Fixes

### PR DESCRIPTION
**Pull request by MBkkt**
### Scope & Purpose

SEARCH-402 Fix GEO_DISTANCE and others
SEARCH-403 Enable EXISTS for inverted indexes

- [ ] :hankey: Bugfix
- [ ] :pizza: New feature
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification

### Checklist

- [ ] Tests
  - [ ] **Regression tests**
  - [ ] C++ **Unit tests**
  - [ ] **integration tests**
  - [ ] **resilience tests**
- [ ] :book: CHANGELOG entry made
- [ ] :books: documentation written (release notes, API changes, ...)
- [ ] Backports
  - [ ] Backport for 3.10: https://github.com/arangodb/arangodb/pull/17269
  - [ ] Backport for 3.9: *(Please link PR)*
  - [ ] Backport for 3.8: *(Please link PR)*

#### Related Information

*(Please reference tickets / specification / other PRs etc)*

- [ ] Docs PR: 
- [ ] Enterprise PR:
- [ ] GitHub issue / Jira ticket:
- [ ] Design document:

### CODE REVIEW

**Review by Dronplane — APPROVED** [review]
LGTM

File: arangod/IResearch/IResearchFilterFactory.cpp [hunk 1]
```diff
@@ -3953,6 +3949,7 @@ frozen::map<irs::string_ref, ConvertionHandler,
     {GEO_INTERSECT_FUNC, fromFuncGeoContainsIntersect},
     {"GEO_IN_RANGE", fromFuncGeoInRange},
     {"GEO_CONTAINS", fromFuncGeoContainsIntersect},
+    // GEO_DISTANCE missing because it's not return boolean
```
Comment (gnusi) [inline arangod/IResearch/IResearchFilterFactory.cpp:3952]: `...it doesn't return...`

Comment (MBkkt) [inline arangod/IResearch/IResearchFilterFactory.cpp:3952]:
```suggestion
    // GEO_DISTANCE missing because it doesn't return boolean
```

**Comment by MBkkt** [comment]
https://jenkins01.arangodb.biz/job/arangodb-matrix-pr/24684/

**Pull request merged**